server/routes/bookVisit/selectVisitorsController.test.ts

```diff
@@ -178,17 +183,11 @@ describe('Select visitors', () => {
           expect(bookerService.getEligibleVisitors).toHaveBeenCalledWith(bookerReference, prisoner.prisonerNumber)
           expect(prisonService.getPrison).toHaveBeenCalledWith(prisoner.prisonId)
 
-          expect(sessionData).toStrictEqual({
-            booker: {
-              reference: bookerReference,
-              prisoners: [prisoner],
-            },
```

Comment:
Unrelated - cleaning up some excess test data. No need to assert value of `booker` as this is now mocked by default for all routes in `server/routes/testutils/appSetup.ts`.